src/test/scala/com/gd/twitteranalytics/util/ApplicationConfigReaderTest.scala

```diff
@@ -0,0 +1,28 @@
+package com.gd.twitteranalytics.util
+
+import com.typesafe.config.ConfigException
+import org.scalatest.FreeSpec
+
+class ApplicationConfigReaderTest extends FreeSpec {
+
+  "Methods tests" - {
```

Comment:
should it be `ApplicationConfigReader`?
could you clarify, where did you get this idea ( "Methods tests") from?

Reply:
This class was generated by IDE with a format ("Methods tests")like this. Updated with class name.